tests/rules/use-consistent-global-store-name.test.ts

```diff
@@ -22,35 +23,75 @@ ruleTester().run(path.parse(__filename).name, rule, {
       import { Store } from '@ngrx/store'
       @Component()
       export class FixtureComponent {
-        constructor(private customName: Store){}
+        constructor(private customName: Store) {}
       }
       `,
       options: ['customName'],
     },
   ],
   invalid: [
-    fromFixture(
-      stripIndent`
-        import { Store } from '@ngrx/store'
-        @Component()
-        export class FixtureComponent {
-          constructor(private somethingElse: Store){}
-                              ~~~~~~~~~~~~~   [${messageId} { "storeName": "store" }]
-        }
+    {
+      code: stripIndents`
+      import { Store } from '@ngrx/store'
+      @Component()
+      export class FixtureComponent {
+        constructor(private readonly somethingElse$: Store) {}
+      }
       `,
-    ),
-    fromFixture(
-      stripIndent`
-        import { Store } from '@ngrx/store'
-        @Component()
-        export class FixtureComponent {
-          constructor(private store: Store){}
-                              ~~~~~   [${messageId} { "storeName": "customName" }]
-        }
+      errors: [
+        {
+          column: 30,
+          endColumn: 44,
+          line: 4,
+          messageId: useConsistentGlobalStoreName,
+          data: {
+            storeName: 'store',
+          },
+          suggestions: [
+            {
+              messageId: useConsistentGlobalStoreNameSuggest as MessageIds,
```

Comment:
Still don't get why TS isn't able to infer this 😅

Reply:
I have no clue; I've searched a solution for it... but yea... 🤯